server/app.js

```diff
@@ -5,7 +5,7 @@ const cookieParser = require("cookie-parser");
 const logger = require("morgan");
 const mongoose = require('mongoose');
 
-const indexRouter = require("./routes/index");
+const langingRouter = require("./routes/landing");
```

Comment:
Can you give this a clearer name for its purpose like authRouter?

Reply:
Changed the name to authRouter